src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+Microsoft.CodeAnalysis.VisualBasic.Syntax.SyntaxNodeRemover.New() -> Void
```

Comment:
I guess I'm confused; did these already ship in 1.2, but only now we're aware we shipped them due to the updated analyzer, or they're actually new?